Sources/SimpleSalesforceSDK/SSSDK.swift

```diff
@@ -21,19 +16,14 @@ public class SSSDK {
 
   // MARK: Representation Properties
 
-  private var host: String?
-  private var redirectUri: String?
-  private var clientId: String?
-  private var clientSecret: String?
+  private var config: SFConfig?
+  private var auth: SFAuth = SFAuth()
 
   private init() {}
 
   private func confirmConfiguration() throws {
-    if host == nil || host == "" ||
-        clientId == nil || clientId == "" ||
-        clientSecret == nil || clientSecret == "" ||
-        redirectUri == nil || redirectUri == "" {
-      throw ConfigurationError.runtimeError("SSSDK not configured yet")
+    if let config = config, !config.isValid {
+      throw SSSDKError.invalidConfigurationError
```

Comment:
It will throw the error even though config has the data.
I think we have to change the logic of config.isValid.